### ISSUE TITLE
image processing

### ISSUE THREAD

**Issue by SnehalSuraj** [SnehalSuraj/IMAGE-CLASSIFICATION#1]
Image Processing 
#COMPANY NAME- CODTECH IT SOLUTIONS
#NAME - SNEHAL SURAJ DAREKAR
#INTERN ID-CT08TGI
#DOMAIN- DATA SCIENCE
#DURATION - 4 WEEKS
# MENTOR - NEELASANTOSH
DESCRIPTION-
This project is a deep learning model built using TensorFlow and Keras to classify handwritten digits (0-9) from the Scikit-Learn Digits dataset. The dataset contains 8x8 grayscale images of digits, which are flattened and used as input features for training a neural network.

The model is trained using a Fully Connected Neural Network (FCNN) with ReLU and Softmax activation functions. It achieves high accuracy in recognizing digits and can be further improved using Convolutional Neural Networks (CNNs).
Programming Language: Python
Libraries:
TensorFlow/Keras (Deep Learning)
Scikit-Learn (Dataset, Preprocessing, Model Evaluation)
NumPy & Pandas (Data Handling)
